.github/workflows/flatpak-builder.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+on:
+    push:
+        tags:
+            - "[0-9]+.[0-9]+.[0-9]+-[0-9]{1}"
+
+jobs:
+    flatpak:
+        runs-on: ubuntu-latest
+        container:
+            image: bilelmoussaoui/flatpak-github-actions:freedesktop-22.08
+            options: --privileged
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
+            - name: "Flatpak Builder"
+              uses: flatpak/flatpak-github-actions/flatpak-builder@v6.1
+              with:
+                  bundle: com.slack.Slack.flatpak
+                  manifest: com.slack.Slack.json
+                  cache-key: flatpak-builder-${{ github.sha }}
+                  arch: "x86_64"
+                  verbose: true
+            - name: Release
+              uses: softprops/action-gh-release@v1
+              with:
+                  files: com.slack.Slack.flatpak
```

Comment:
What is the point of this when flathub has a buildbot already??

Reply:
Reverted. Sorry, I was testing some stuff and this wasn't meant to be here.